content/basics/ably.textile

```diff
@@ -0,0 +1,86 @@
+---
+title: What is Ably?
+meta_description: "Ably is a realtime platform delivering powerful digital realtime experiences through a publish/subscribe network."
+---
+
+Ably’s realtime platform provides a scalable, fault-tolerant and worldwide publish/subscribe(pub/sub) network. Providing SDKs in a variety of programming languages with both REST and realtime interfaces Ably helps you get connected.
```

Comment:
This should align with our core messaging. 
"Ably is the platform to power synchronized digital experiences in realtime. Ably's SDKs and APIs power realtime capabilities like Chat, Data Broadcast, Data Synchronization, Multiplayer Collaboration, and Notifications."